### PR TITLE
zhongturtle term proposal

### PR DESCRIPTION
**Pull request by zivzone**

### CODE REVIEW

File: term_proposal/zhongturtle/proposal.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Crude-NN
+## Introduction
+- It's a neural network library implemented from scatch, and most of them is basic components for building a complete model. Starts from layers by layers, for example, convolution layer and fully connected layer, and fuctions by fuctions, for example, relu , pooling , and sigmoid.
+- I will use the functions I made to build the deep learning model for several basic task to prove my library is work, such as mnist classification, cifar-10 classficaion and etc. You can see more in "Case/Application" section.
+- After I can build the model with my library, I will try to do some visulize and anaylisis, like show the loss curve ,calculate time and space complexity. You can see more in "Visualize" & "Analyze" sections.
+- We hope in the future this library is not only for single machine, but also surport distributed system.  
+- Although it only supports python now, and hope more in the future
+- Powered by Velitus.tech
+## TO-DO List
```
Comment (yungyuc) [inline term_proposal/zhongturtle/proposal.md:9]: You have many items in the list.  It seems to be too long if you want to do them all in the 2 months.  And be reminded that you need to demonstrate what you learn about software engineering during the course, which takes non-trivial effort.

Could you prioritize the todo items, before you move forward for the implementation?

**Review by yungyuc — CHANGES_REQUESTED** [review]
The proposal is clear about what you want to do.  As long as you give a better scope about the project extent it's good to go.

**Comment by yungyuc** [comment]
@zhongturtle before this can be merged, you need to change the target branch to be `term-proposal`, instead of `master`.

**Comment by yungyuc** [comment]
@zhongturtle any update?

**Comment by yungyuc** [comment]
@zhongturtle you need to merge against `term-proposal` branch, not `master`.

**Comment by zivzone** [comment]
@yungyuc  Sorry for the late reply, I have re-PR and add the priority of action items

**Comment by yungyuc** [comment]
Thanks.  You are still merging against `master`.  It's not correct.  The PR should be against `term-proposal`.

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
Merged https://github.com/yungyuc/nsdhw_19au/pull/72 instead.

**Pull request closed without merging**